tutorial_notebooks/pauli_lindblad_per/ProcedureOverview.ipynb

```diff
@@ -935,7 +935,7 @@
     "noise_strength = 0\n",
     "insts = [inst for inst in perexp._per_runs[0].instances if inst.noise_strength == 0]\n",
     "expectations = [inst.get_adjusted_expectation(QiskitPauli(\"IX\")) for inst in insts]\n",
-    "print(expectations)"
+    # "print(expectations)"
```

Comment:
We can just comment this our or replace with a histogram plot. 
The very long list of numbers is uncomfortable when reading through the notebook.